client/src/components/WebAuthn/Verify.js

```diff
@@ -1,7 +1,9 @@
 /* global window */
 
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import { base64ToByteArray, byteArrayToBase64 } from 'lib/convert';
+import publicKeyType from '../../types/publicKey';
```

Comment:
You can use types/... without the relative path

Reply:
Oh, didn't know. Cool. Ta.